Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```
@@ -767,7 +778,7 @@
         Caustic: 1
   - type: MultiHandedItem
   - type: Item
-    sprite: Mobs/Pets/smile.rsi
+    sprite: Mobs/Pets/smile/smile.rsi
```

```suggestion
    sprite: Mobs/Pets/Smile/smile.rsi
```

folder naming convention